server/lib/src/server/mod.rs

```diff
@@ -672,6 +673,7 @@ pub trait QueryServerTransaction<'a> {
                             OperationError::InvalidAttribute("Invalid uihint syntax".to_string())
                         }),
                     SyntaxType::AuditLogString => Ok(PartialValue::new_utf8s(value)),
+                    SyntaxType::EcKey => Ok(PartialValue::Iutf8(value.to_string())),
```

Comment:
I don't think we need to be able to query these keys. Why would we need to look up a key by it's ID? We only need to lookup the users spn/name, and the key is associated.